md-setup/src/main/java/com/wipro/ats/bdre/md/setup/SetupDB.java

```diff
@@ -42,9 +41,14 @@
     private SessionFactory sessionFactory;
     private Session session;
     private static java.util.Properties map;
+    private static Pattern pattern = Pattern.compile("\\$\\{(?<key>[^}]*)\\}");
+    String inFile="In File: ";
+    String badLine="; Bad Line: ";
+    String lineH="Line #";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Line #" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)